### ISSUE TITLE
Unable to set "Allow discussion" to "yes"

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1797]
steps to reproduce:

- Create a new folder
- edit it, set "Allow discussion" to "yes"
- edit again, now it's back to "No"

In the network requests I can see that the patch request is sent as `allow_discussion: "True"`. Perhaps it's a wrong widget? Wrong vocabulary values?

**Comment by nileshgulia1** [comment]
ref #1795. @tiberiuichim I can look into this.

**Comment by nileshgulia1** [comment]
@tiberiuichim I think its nothing related to widget, as on `GET_CONTENT` action the `allow_discussion` field is always returning ```false```, even in the backend its set to `true`.
Also in schema `allow_discussion` field is not exposing a default property while others(like `exclude_from_nav`) do.

**Comment by tiberiuichim** [comment]
@nileshgulia1 Indeed, I've noticed that as well. Perhaps the answer is in plone.restapi then...
